server.js

```diff
@@ -0,0 +1,115 @@
+/*jshint esversion: 6 */
+
+var express = require('express');
+var bodyParser = require('body-parser');
+var app = express();
+
+var buzzwords = {
+  buzzWords: [],
+};
+
+var currentScore = {
+  'success': null,
+   newScore: 0,
+};
+
+app.use(express.static('public'));
+
+app.use(bodyParser.urlencoded({ extended: false}));
+app.use(bodyParser.json());
+
+app.get('/', (req,res) => {
+  res.render('index');
+});
+
+app.get('/buzzwords', (req,res) => {
+  res.send(buzzwords);
+});
+
+app.post('/buzzwords', (req, res) => {
+  var checkBuzzword = req.body.buzzword;
+  var currentArray = buzzwords.buzzWords;
+  var test = currentArray.filter(filterBuzzwords);
+
+  if(test.length > 0){
+    res.send({ "success": false });
+  }
+
+  if(test.length === 0){
+  var score = req.body.score;
+  var convertToNumber = Number(score);
+  req.body.score = convertToNumber;
+  buzzwords.buzzWords.push(req.body);
+  res.send({ "success": true });
+  }
+
+  function filterBuzzwords(item){
```

Comment:
This function is being redefined in multiple places. It's probably a good idea to pull this out to a higher scope so you'll only need to define it once.